### ISSUE TITLE
Multiple Console Errors

### ISSUE THREAD

**Issue by varundhand** [ashutosh1919/masterPortfolio#333]
There are multiple **syntax**, **semantic** and **logic errors** throughout the website which needs to be fixed in order to a clean, error-free **console**.

## Screenshots:

![Screenshot 2023-10-25 at 2 03 18 AM](https://github.com/ashutosh1919/masterPortfolio/assets/110025628/5afc51b5-b72c-4e2d-af7f-7bf1b18ae69e)

## Note to Reviewer:

I'd like to work on this issue and raise a PR addressing these issues :)

**Comment by ashutosh1919** [comment]
@varundhand Yes unfortunately we never got to the part to fix that. Thanks for noticing it. Feel free to raise PR to fix this.

**Comment by varundhand** [comment]
I ll issue the PR right away!

**Comment by varundhand** [comment]
I have already raised the PR @ashutosh1919 but I mentioned the wrong issue in it. I fixed it now. Here's the PR: #334
